server/models/postgis/task_annotation.py

```diff
@@ -12,17 +12,19 @@ class TaskAnnotation(db.Model):
     task_id = db.Column(db.Integer, nullable=False)
     annotation_type = db.Column(db.String, nullable=False)
     annotation_source = db.Column(db.String)
+    annotation_markdown = db.Column(db.String)
     updated_timestamp = db.Column(db.DateTime, nullable=False, default=timestamp)
     properties = db.Column(db.JSON, nullable=False)
 
     __table_args__ = (
         db.ForeignKeyConstraint([task_id, project_id], ['tasks.id', 'tasks.project_id'], name='fk_task_annotations'), db.Index('idx_task_annotations_composite', 'task_id', 'project_id'), {})
 
-    def __init__(self, task_id, project_id, annotation_type, annotation_source, properties):
+    def __init__(self, task_id, project_id, annotation_type, annotation_source, annotation_markdown, properties):
```

Comment:
Can this be refactored as:

__init__(self, task_id, project_id, annotation_type, properties, annotation_source=None, annotation_markdown=None): 

To keep the requried parameters first and the nullable as optionals with None as default value?